docs/docs/docs/getting-started/installation.md

```
@@ -41,6 +41,35 @@ You will need to configure the API to work correctly.
 | `MINIO_ROOT_PASSWORD`                  | Root password for MinIO, used for securing administrative access to MinIO object storage.                             |
 | `POSTGRES_PASSWORD`                    | Password for the PostgreSQL database (Docker Compose), used for database authentication and security.                 |
 
+## Running the Setup Script
+To configure the .env file, run one of the following commands in your project’s root directory:
+
+```bash
+npm run setup
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)